README.md

```diff
@@ -50,6 +50,23 @@ Build one package
     SRCRPM=$(rpmbuild -bs --nodeps --define '_with_test 1' --define "tag $GITTAG" *.spec | awk '{print $2}')
     mock -r epel-7-x86_64-openio-sds-unstable --define '_with_test 1' --define "tag $GITTAG" --rebuild $SRCRPM
 
+Repair the mirror / Remove a broken package
+--------
+
+    # Log on the build VM (OpenStack)
+    ssh buildsys-rpm
+    # Go inside the mirror (NFS mounted directory from mirror2.openio.io)
+    cd /mnt/koji/mirror/pub/repo/openio/sds/17.04/el/7/x86_64
+    # Remove the broken package(s)
+    sudo rm -i openio-gridinit-1.7.0*
+    # Re-create the metadatas
+    createrepo .
```

Comment:
`--update` will make this faster.

Reply:
Yep, but I want to recreate everything from scratch here.

I'll use `--update` in production, assuming it will work OK, but for a repair procedure I prefer to take the safer way.